packages/x-components/src/x-modules/history-queries/store/__tests__/actions.spec.ts

```diff
@@ -369,9 +391,9 @@ describe('testing history queries module actions', () => {
     });
 
     // eslint-disable-next-line max-len
-    it('updates a history query if search response change because a filter is selected', async () => {
+    it('updates a history query when the search response changes because a filter is selected', async () => {
       gato.totalResults = 50;
-      const selectedFilters = Object.values(requestFilters)[0];
+      const selectedFilters = createHistoryQueriesFiltersList(responseFacets);
```

Comment:
You're asking the function that you are testing what it should return. `selectedFilters` should be a hardcoded value of what the history query selected filters should be given the request you're dispatching the action with.